benchmarks/continous/db/tool/orm/src/models.rs

```diff
@@ -19,8 +25,7 @@ pub struct NewFtTransfer {
     pub disjoint_workloads: bool,
     pub num_shards: i32,
     pub num_unique_users: i32,
-    // TODO next to fields should be i64 (and sql bigint)
-    pub size_state_bytes: i32,
+    pub size_state_bytes: i64,
     pub tps: i32,
-    pub total_transactions: i32,
+    pub total_transactions: i64,
```

Comment:
Why we use signed integers here? Postgres cannot store u64?

Reply:
Apparently PostgreSQL does not support unsigned integers and they are not part of the SQL standard [[ref](https://stackoverflow.com/a/20850942)]. The benchmarks db is very lightweight, so I think we don't need to worry about the overhead of storing signed integers.